lib/haml_lint/reporter.rb

```diff
@@ -5,12 +5,15 @@ module HamlLint
   # @abstract
   class Reporter
     attr_reader :lints
+    attr_reader :files
 
     # @param logger [HamlLint::Logger]
     # @param report [HamlLint::Report]
-    def initialize(logger, report)
+    # @param files  [Array]
+    def initialize(logger, report, files)
       @log = logger
       @lints = report.lints
+      @files = files
```

Comment:
It's being used [here](https://github.com/skateman/haml-lint/commit/10d89c63581dc4d9e3001370daeb909a66e9e1b5#diff-202b23a957ac37cd746eb7ec9840c680R17), but it's in a separate commit. The `lints` shows only the files with issues if I'm right, but correct me if not!